bin/mr_bump

```diff
@@ -55,6 +55,12 @@ loop do
     MrBump.file_prepend('CHANGELOG.md', md_changes)
     system 'nano CHANGELOG.md'
     loop do
+      log = File.open('CHANGELOG.md', 'r').read
+      md_changes = log.split("\n\n").first.split("\n").drop(1)
```

Comment:
Im sure there is a better way of doing this, @xulaus Any idea? There must be something that does not involve going through the full file, just picking the text between the first two `#`?